src/pykeen/datasets/ea/wk3l.py

```diff
@@ -102,6 +102,7 @@ def _load_df(self, key: Union[None, EASide, Tuple[EASide, EASide]], **kwargs) ->
             engine="python",
             encoding="utf8",
             dtype=str,
+            keep_default_na=False,
```

Comment:
so this was sufficient? because this won't solve the issue if there are empty entries

Reply:
empty entries are read as `nan`.

Reply:
so even if there's an empty string it gets the string `"nan"`? 

maybe you should still do dropna() in addition to `keep_default_na=False,`